contracts/libs/Types.sol

```diff
@@ -33,11 +33,20 @@ library Types {
         bool withdrawn;
     }
 
+    struct Submission {
+        bytes txs;
+        bytes32 updatedRoot;
+        uint256[2] signature;
+        uint256 tokenType;
+    }
```

Comment:
Add Submission struct to prevent stack too deep in the submitBatch.

Reply:
Can we just use commitments struct?

Reply:
I considered commitments struct too, but the commitment has extra fields that are set inside the function body of submitBatch. Using the commitment struct:
1. waste extra zero fields on the input.
2. confuses the developer because you send input that has no effect.

Reply:
Gotcha, cool let's roll with this.